service/handlers.go

```diff
@@ -88,19 +88,22 @@ func handleQuery(c *Core, w *ResponseWriter, r *Request) {
 		select {
 		case <-timer.C:
 			if err := writer.WriteProgress(meter.Progress()); err != nil {
+				w.Logger.Warn("Error Writing Progress", zap.Error(err))
```

Comment:
Capitalize just the first word.
```suggestion
				w.Logger.Warn("Error writing progress", zap.Error(err))
```